api/proto/teleport/legacy/types/types.proto

```diff
@@ -6159,6 +6161,57 @@ message PluginSCIMSettings {
   string default_role = 2;
 }
 
+// PluginAWSICSettings holds the settings for an AWS Identity Center integration.
+message PluginAWSICSettings {
+  option (gogoproto.equal) = true;
+
+  // IntegrationId is the Teleport OIDC integration used to gain access to the
+  // AWS account
+  string integration_id = 1;
+
+  // InstanceRegion is the region the target Identity Center instance is
+  // configured in
+  string instance_region = 2;
+
+  // InstanceARN is the arn of the Identity Center instance to manage
+  string instance_arn = 3;
+
+  // Provisioning holds settings for provisioing users and groups into AWS
+  PluginAWSICProvisioningSettings provisioning = 4;
+}
+
+// PluginAWSICProvisioningSettings hold settins provisioning-specific settings
+message PluginAWSICProvisioningSettings {
+  option (gogoproto.equal) = true;
+
+  // BaseURL is the SCIM base URL
+  string base_url = 1;
+
```

Comment:
Don't we also need a field to store SCIM token once the plugin retrieves it from the `PluginCredentialsV1` ?

Reply:
Yeah, but this should be stored separately as a static credential. I'll add a PluginStaticCredentialRef to represent it.